### PR TITLE
Use json read file

### PR DESCRIPTION
**Pull request by wobh**

### CODE REVIEW

**Comment by verdammelt** [comment]
@wobh unless I'm reading these pull-requests wrong it looks like each one your submitted includes all the changes from the previous? Probably not what you intended?

**Comment by wobh** [comment]
Oops.  I'll have to figure it out a bit later. Thanks!

**Comment by canweriotnow** [comment]
@wobh It looks like you branched off from the previously submitted PR branch each time? Or you had already merged into local master? Solution is to either a) use a single branch, subsequent pushes should get added to existing PR, or b) create each new PR branch off of master.

No big deal though, haven't seen nay conflicts as I've merged them.

File: exercism.el [hunk 1]
```diff
@@ -35,22 +36,14 @@
...
-    (buffer-string)))
-
-(defun get-exercism-config (file-path)
+(defun exercism-get-config (file-path)
   "Parses exercism json config into a plist."
-  (let ((json-object-type 'plist)
-        (json-string (get-string-from-file file-path)))
-    (json-read-from-string json-string)))
+  (let ((json-object-type 'plist)) 
+    (json-read-file file-path)))
```
Comment (canweriotnow) [inline exercism.el:46]: Ah! Dunno how I missed this fun when I was reading the json docs. Much cleaner.

**Pull request merged**